axelrod/strategies/geller.py

```diff
@@ -40,7 +40,7 @@ class Geller(Player):
 
     name = 'Geller'
     classifier = {
-        'memory_depth': -1,
```

Comment:
FYI: This one I think we did because "it looks into the future" (not suggesting we change it back necessarily).